<ui version="4.0" stdsetdef="1" >
  <class>QAxSelect</class>
  <widget class="QDialog" name="QAxSelect" >
    <property name="objectName" >
      <string notr="true" >QAxSelect</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>326</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Select ActiveX Control</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item rowspan="2" row="0" column="1" colspan="1" >
        <layout class="QVBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="objectName" >
                <string notr="true" >buttonOk</string>
              </property>
              <property name="text" >
                <string>OK</string>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="objectName" >
                <string notr="true" >buttonCancel</string>
              </property>
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="0" column="0" >
        <widget class="QListView" name="ActiveXList" >
          <property name="objectName" >
            <string notr="true" >ActiveXList</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <layout class="QHBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="TextLabel1" >
              <property name="objectName" >
                <string notr="true" >TextLabel1</string>
              </property>
              <property name="text" >
                <string>COM &amp;Object:</string>
              </property>
              <property name="buddy" stdset="0" >
                <cstring>ActiveX</cstring>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="ActiveX" >
              <property name="objectName" >
                <string notr="true" >ActiveX</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <includes>
    <include location="local" >qaxwidget.h</include>
  </includes>
</ui>
